### ISSUE TITLE
Create mockups for home page redesign

### ISSUE THREAD

**Issue by Saltzshaker** [popuparchive/pop-up-archive#1411]

**Comment by Saltzshaker** [comment]
Two layouts discussed w/ Anne + Emily:

1. Page that would link to user stories on a separate page, and get rid of standalone 'about' page, condensing it into a carousel. Also, mocked-up single row logos, with @annewootton's idea that this row would have an animation refreshing with the rest of the logos
![mockup1](https://cloud.githubusercontent.com/assets/8497203/14374260/c75f488a-fd08-11e5-9903-a7cba73bef96.png)

2. Basically the same besides modified logos and user stories (@annewootton and I discussed other user story changes, but pasted original here as a placeholder while we finalize design)
![mockup2](https://cloud.githubusercontent.com/assets/8497203/14374271/d29fe9fc-fd08-11e5-9eb4-a57cd9324a1a.png)

**Comment by annewootton** [comment]
Looks great. For the second option, would the "About" page need to change
too, given that we both thought it merits updating? (The first option
accounts for this by condensing "About" onto the homepage.)

**Comment by Saltzshaker** [comment]
@annewootton yeah, if we went with the second option, we would decide on what changes to make to the about page separately, and i could mock something up for that at that point — probably similar format, maybe cutting down on some of the redundant/fluffy text, getting higher res pics, rethinking the video, etc. as we've talked about

**Comment by annewootton** [comment]
Cool. I defer to @baileyspace now to chime in on which option she prefers.

**Comment by baileyspace** [comment]
Looks good to me.

**Comment by Saltzshaker** [comment]
@baileyspace option 1?

**Comment by baileyspace** [comment]
Yes, option 1.
